clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -264,3 +264,154 @@
   CmakeSyntax: add_executable_opts
   In: add_executable(${value} OPTIONS ${options})
   Out: add_executable(${value})
+
+- Rule: rule_target_link_libraries_CUDA_LIBRARIES
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: CUDA_LIBRARIES_target_link_libraries
+  In: target_link_libraries(${CULibraries})
+  Out: target_link_libraries(${CULibraries})
+  Subrules:
+    CULibraries:
+      In: |
+        \${CUDA_LIBRARIES}
```

Comment:
Pls add comments why ${CUDA_LIBRARIES} should be removed.

Reply:
Added. Done.